### DISCUSSION TITLE
Feature request : support tailwind prefix

### DISCUSSION THREAD

**Discussion by snimavat** [konstaui/konsta#42] [Q&A]
### Clear and concise description of the problem

We have prefix `t-` for tailwind utilities. but Konsta wouldnt pick tht automatically and adds so it uses tailwind without prefix
resulting in none of the components styles working 

### Suggested solution

None

### Validations

- [X] Follow our [Code of Conduct](https://github.com/tailwind-mobile/tailwind-mobile/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://tailwind-mobile.com).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [ ] I'm willing to open a PR

**Comment by snimavat** [comment]
Is it supported ?

**Comment by nolimits4web** [comment]
At the moment not supported

**Reply by snimavat** [reply]
This is a deal breaker, i believe lost of people use prefix for tailwind
Would have been great to use konsta.

Hope to see the prefix support soon,

**Comment by snimavat** [comment]
@nolimits4web cant we keep this as an issue, this is a feature request rather thn a question or discussion.

**Comment by snimavat** [comment]
Any update on this one @nolimits4web ?
